automation/taskcluster/lib/tasks.py

```diff
@@ -512,20 +533,3 @@ def schedule_task(queue, taskId, task):
     result = queue.createTask(taskId, task)
     print("RESULT", taskId)
     print(json.dumps(result))
-
-
-def schedule_task_graph(ordered_groups_of_tasks):
```

Comment:
Okay to remove this function, for now. I'd lean towards this solution https://github.com/mozilla-mobile/reference-browser/pull/681#discussion_r272609457, in a follow up